src/sql/workbench/parts/jobManagement/browser/media/jobHistory.css

```diff
@@ -151,40 +151,40 @@ input#accordion:checked ~ .accordion-content,
 	content: url("back_inverse.svg");
 }
 
-.vs .action-label.icon.newStepIcon {
+.vs .action-label.codicon.newStepIcon {
```

Comment:
So this is what you were saying with all actions being from vscode so having codicon instead of icon even if they aren't using the codicon font?

Reply:
Yeah so anything that inherits from Action class will get a CSS class "codicon" included, where previously it was "icon". Hence most of this CSS stopped getting applied and icons weren't showing up. 

Note an alternative is not to rely on the `.codicon`/`.icon` at all and just match `.action-label.newStepIcon`. I'd prefer to play it safe and do the direct changes instead.